recipes/arrays.md

```diff
@@ -64,7 +64,7 @@ items.unshift(42);
 ```dart
 final length = items.length;
 
-final newItems = items..addAll(otherItems);
+final newItems = items.addAll(otherItems);
```

Comment:
cascading is required to be consistent with `js` version. `.addAll()` is void, that's why I've used cascading

Reply:
I think you don't need cascade here, because cascades usually call when you need to call one more void methods. I think, you need to add additional receipt for describing it.